frontend/src/Components/Conditions/ConditionsMap.tsx

```diff
@@ -413,17 +413,13 @@ const ConditionsMap = (props: any) => {
         </div>
         <div className="filter-container">
           <div className="input-selector-container">
-            <select
-              className="input"
-              defaultValue={mode}
-              onChange={inputChange}
-            >
-              {conditionTypes.map((value) => (
+            <MultiSelector
+              onChange={conditionTypes.map((value) => (
```

Comment:
I get this error, not sure why it cannot be assigned :///

Type 'Element[]' is not assignable to type '(value: Element[]) => void'.
  Type 'Element[]' provides no match for the signature '(value: Element[]): void'

Reply:
@Seb-sti1 If you have time, could you be so kind and let me know what's up and how to fix? Have been going through stackExchange but still no luck finding a fix.

Reply:
Not sure what you are trying to do here, but `onChange` probably is only a callback function returning the selected values, so do maybe:
```ts
onChange={callback(value) => {
   console.log("Selected values", value);
}
```

And then to add the children, add a `children` in the props of the `MultiSelector` (like for the [MapWrapper](https://github.com/DTU-SE-Group-D/LiRA-Viz/blob/dev/frontend/src/Components/Map/MapWrapper.tsx#L17))

```ts
<MultiSelector
  onChange={callback(value) => {
     console.log("Selected values", value);
  }>
{conditionTypes.map((value) => (<option value={value} key={value}>
      {value}
    </option>
  ))}
</MultiSelector>
```